project-base/app/config/paths.yml

```diff
@@ -33,3 +33,4 @@ parameters:
     shopsys.vendor_dir: '%kernel.root_dir%/../vendor'
     shopsys.js_form_validator_output_vendor_dir: '../vendor'
     shopsys.data_fixtures_images.resources_dir: '%shopsys.data_fixtures.resources_dir%/images/'
+    shopsys.elasticsearch.structure_dir: '%shopsys.resources_dir%/definition/'
```

Comment:
is there a way to move this to framework? Mainly because of upgradeability.

Reply:
In my opinion there will be much more changes from project view than from framework view and until we have some solution that these will be extendable I would prefer to keep them in project-base.

Reply:
Yeah you are propably right, I saw it from perspective of upgrading to this version where you will be forced to create some json file defining structure, but it is true that this will be changed often so it makes sense.